master/master.go

```diff
@@ -112,6 +118,32 @@ func (master *Master) Start(configuration *Config) {
 	//master.agent.SetContextNotifyHandler(master.onReceiveContextNotify)
 	master.agent.SetContextAvailabilityNotifyHandler(master.onReceiveContextAvailability)
 
+	 go func() {
+	      
+              body, err := json.Marshal(map[string]string{
+                       "status" : "Master is Up"})
+                if err != nil {
+                        fmt.Println(err)
+                }
+               //fmt.Println(master.cfg.HTTPS)
+               master.cfg.HTTPS.LoadConfig()
+               client := master.cfg.HTTPS.GetHTTPClient()
+               fmt.Println("==== client =====",client)
+               req2, err := http.NewRequest("POST", master.designerURL+"/masterNotify", bytes.NewBuffer(body))
+               fmt.Println("++++++ req ++++++ and err",req2,err)
+               for {
+                        //resp, err := client.Post(url,"application/json" , bytes.NewBuffer(body))
+                       time.Sleep(5 * time.Second)
```

Comment:
need to define a variable retryInterval and set the value 5. and use retryInterval variable here.